lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
Yes, tried the lib folder for some reason this did not appear to work, the (mysql) driver never got initialized for entries in resources.xml with jar in lib folder but did with the tomee/lib folder. Will check once more though.